src/cluster/websocket/WebSocket.h

```diff
@@ -228,6 +228,8 @@ class WebSocketEventDispatcher {
 
 
     void HandleSubscriptions(WebSocketClientEntry& entry, std::string_view buf);
+    // All subscriptions are now active, raise the websocket_client_added event and send the ack.
+    void HandleSubscriptionsActive(WebSocketClientEntry& entry);
```

Comment:
"Now" doesn't work well in a docstring since it could be run at any time.

Maybe something like

```suggestion

    // For all currently active subscriptions of the entry, raise the internally `websocket_client_added` event, and send the ack to the client.
    void HandleSubscriptionsActive(WebSocketClientEntry& entry);

```

Probably nit, but since we add a docstring I would also surround this by empty lines.